src/platform/x11.rs

```diff
@@ -171,6 +172,9 @@ pub trait WindowBuilderExtX11 {
     fn with_x11_visual<T>(self, visual_infos: *const T) -> Self;
 
     fn with_x11_screen(self, screen_id: i32) -> Self;
+    #[cfg(feature = "x11")]
+    /// Build window with X11's parent window. Only relevant on X11.
```

Comment:
We're inside the X11 extension.

```suggestion
    /// Build window with parent window.
```

Reply:
thanks! fixed.